src/main/paradox/docs/api/assets/realms/realm-added-ref.json

```diff
@@ -0,0 +1,17 @@
+{
+  "@context": "https://bluebrain.github.io/nexus/contexts/realms.json",
+  "@id": "http://dev.nexus.ocp.bbp.epfl.ch/auth/realms/realm1",
+  "@type": "nxv:realm",
+  "name": "realm1",
+  "openid-configuration": "http://dev.nexus.ocp.bbp.epfl.ch/auth/realms/bbp-test/.well-known/openid-configuration",
```

Comment:
please replace all references of `http://dev.nexus...` with `http://nexus.example.com`